server/webhook_parser.go

```diff
@@ -304,6 +304,7 @@ func appendCommentNotifications(wh *webhook, verb string) {
 		message:       fmt.Sprintf("%s **commented** on %s:\n>%s", commentAuthor, jwh.mdKeySummaryLink(), jwh.Comment.Body),
 		postType:      PostTypeComment,
 		commentSelf:   jwh.Comment.Self,
+		recipientType: recipientTypeAssignee,
```

Comment:
@mickmister reply on this comment https://github.com/mattermost/mattermost-plugin-jira/pull/810#discussion_r760752142 

just to make it generalise the recipientType values as constant considering will be used further
if you want can revert the changes

Reply:
> considering will be used further

Are you saying this value is used elsewhere? Also do we need a `recipientTypeMention`?

Reply:
What is the value used for? Is it checked somewhere in a different PR?

Reply:
i dont see any where it is used , this was a hardcoded value for `webhookUserNotification` so created constant for that and used.